mathics/builtin/specialfns/hypergeom.py

```diff
@@ -8,18 +8,20 @@
 
 import mpmath
 import sympy
-from mathics.core.convert.mpmath import from_mpmath
-from mathics.core.convert.sympy import from_sympy
+
 from mathics.core.attributes import (
     A_LISTABLE,
     A_N_HOLD_FIRST,
     A_NUMERIC_FUNCTION,
     A_PROTECTED,
     A_READ_PROTECTED,
 )
-from mathics.core.builtin import SympyFunction, MPMathFunction
+from mathics.core.builtin import MPMathFunction, SympyFunction
+from mathics.core.convert.mpmath import from_mpmath
+from mathics.core.convert.sympy import from_sympy
```

Comment:
The following imports are needed in the suggested change below. Also, they would appear _after_ the next line (but that isn't part of changed lines so I don't know how to add this suggestion there.)

```suggestion
from mathics.core.convert.sympy import from_sympy
from mathics.core.number import FP_MANTISA_BINARY_DIGITS
from mathics.eval.arithmetic import run_mpmath
```

```